### PR TITLE
Style weight and remove FA icon from link text

### PR DESCRIPTION
**Pull request by eselkin**
## Description

relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14479
Uses va-link component for link, uses style from storybook for alert (h2, headline, div, p).

## Testing done & Screenshots

Before:
Screenshot 2023-09-06 at 12 31 18 PM

Screenshot 2023-09-06 at 12 37 21 PM

After:
http://57625561fb997654f1271ea7d10c946f.review.vetsgov-internal/lovell-federal-health-care-va/
Screenshot 2023-09-06 at 5 24 27 PM

http://57625561fb997654f1271ea7d10c946f.review.vetsgov-internal/lovell-federal-health-care-tricare/
Screenshot 2023-09-06 at 5 24 22 PM

## QA steps

1. Do this
   - [ ] Go to Lovell TriCare and VA pages
2. Then validate Acceptance Criteria from issue

## Acceptance criteria

- [x] I expect that the switch link on [Lovell pages](https://www.va.gov/lovell-federal-health-care-tricare/) uses the [warning alert style](https://design.va.gov/components/alert#warning-alert)
- [x] I expect that the switch link fonts, size, styling align with design system
- [x] I expect the font weight of active link is set to 700 (done by va-link active component)
- [x] I expect that the tag is ended before the space between beneficiary and > (handled by va-link active component)
- [x] I expect that the link to the updates in staging have been provided in this ticket to @aklausmeier for review
- [x] @aklausmeier has approved updates

## Definition of done
- [x] Events are logged appropriately (No changes to events)
- [x] Documentation has been updated, if applicable (N/A)
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by randimays — APPROVED** [review]
LGTM!

File: src/site/includes/lovell-switch-link.drupal.liquid [hunk 1]
```diff
@@ -8,8 +8,15 @@
   {% endif %}
 
   <va-alert status="warning" class="vads-u-margin-bottom--2" id="va-info-alert">
-    <div class="vads-u-margin-top--0">
-      <strong>You are viewing this page as a {{ currentPageVariation }} beneficiary.</strong> <a href="{{ entityUrl.switchPath }}">View this page as a {{ switchPageVariation }} beneficiary <i aria-hidden="true" class="fas fa-angle-right"></i></a>
-    </div>
+    <h2 slot="headline">You are viewing this page as a {{ currentPageVariation }} beneficiary.</h2>
+    <div>
```
Comment (maxx1128) [inline src/site/includes/lovell-switch-link.drupal.liquid:12]: **Question/Nitpick:** is this extra `div` wrapper needed or part of any documented markup? I tried removing it and the layout and margin for the component barely changed.

Screen Shot 2023-09-07 at 11 07 37 AM

**Review by maxx1128 — APPROVED** [review]
Looks good on my computer. One small note on the markup but it's not a merge blocker.

**Review by aklausmeier — APPROVED** [review]
Looks great, thank you

File: src/site/includes/lovell-switch-link.drupal.liquid (hunk 1 above)
Comment (eselkin) [reply src/site/includes/lovell-switch-link.drupal.liquid:12]: It does nothing, but in adhering to the guidance in the storybook, I put it in.

**Pull request merged**